_includes/modules/ResizeMatrix.md

```diff
@@ -22,7 +22,7 @@ This module lets the user resize their matrix to the desired size.
 
 **Detailed Description**
 
-This module lets the user resize their matrix to the desired size. In the GUI, one can choose the size of the matrix (by defining the number of rows and columns). If the number of elements in the original matrix is not equal to the (no. of column)*(no. of rows) then the new matrix would have zeros appended to the new matrix or few of the elements truncated from their previous matrix depending on the size of the new matrix that they set. They also have an option to set the major of the matrix to row major and column major. 
+This module lets the user resize their matrix to the desired size. In the GUI, one can choose the size of the matrix (by defining the number of rows and columns). If the number of elements in the original matrix is less than the (no. of column)X(no. of rows) then the new matrix would have zeros appended to it. And if the number of elements in the original matrix are more than the (no. of column)X(no. of rows) then few of the elements will be truncated from original martix. Users can also choose between the row or columns major ordering of the resizing.
```

Comment:
change "then the new matrix would have zeros appended to it" to "then the new matrix will be padded with zeros"

remove "And" in next sentence

change "then few of the elements will be truncated from original martix" to "then the matrix will be truncated"